Sources/Hummingbird/Files/FileMiddleware.swift

```diff
@@ -98,12 +98,12 @@ public struct FileMiddleware<Context: BaseRequestContext, Provider: FileProvider
 
             // Remove percent encoding from URI path
             guard let path = request.uri.path.removingPercentEncoding else {
-                throw HTTPError(.badRequest)
+                throw HTTPError(.badRequest, message: "Invalid URL")
             }
 
             // file paths that contain ".." are considered illegal
             guard !path.contains("..") else {
-                throw HTTPError(.badRequest)
+                throw HTTPError(.badRequest, message: "URLs including '..' are not supported")
```

Comment:
I don't think we should provide a message here, this is basically just used maliciously. Any info we give can only help attackers.

Reply:
removed